padd.sh

```diff
@@ -104,6 +104,9 @@ ConstructAPI() {
 	# If no arguments were supplied set them to default
 	if [ -z "${URL}" ]; then
 		URL=127.0.0.1
+        # when no $URL is set we assume PADD is running locally and we can get the port value from FTL directly
+        PORT="$(pihole-FTL  --config webserver.port)"
```

Comment:
Since you are already changing the code... there is a extra space:

```suggestion
        PORT="$(pihole-FTL --config webserver.port)"
```

Reply:
Done.